### PR TITLE
Chat update

### PR DESCRIPTION
**Pull request by piotrdrag**

### CODE REVIEW

File: chat/xmpp.properties [hunk 1]
```diff
@@ -28,15 +28,38 @@ connection.error.failedToGetAResource=Pobieranie zasobu nie powiodło się
 conversation.error.notDelivered=Następująca wiadomość nie mogła zostać dostarczona: %S
 
 conversation.error.joinFailed=Nie można dołączyć do: %S
+conversation.error.joinForbidden=Nie można dołączyć do pokoju „%S”, ponieważ zbanowano Cię w nim.
```
Comment (splewako) [inline chat/xmpp.properties]: Don't use "Cię".

File: chat/irc.properties [hunk 2]
```diff
@@ -80,24 +78,24 @@ message.whoisEntry=\ua0\ua0\ua0\ua0%1$S: %2$S
 message.unknownNick=„%S” nie jest znanym pseudonimem użytkownika.
 message.channelKeyAdded=Użytkownik %1$S zmienił hasło dla kanału na „%2$S”.
 message.channelKeyRemoved=Użytkownik %S usunął hasło kanału.
-message.banMasks=Użytkownicy łączący się z następujących adresów są niedopuszczani do „%S”:
-message.noBanMasks=Brak wykluczonych adresów dla „%S”.
-message.banMaskAdded=Użytkownicy łączący się z adresów pasujących do „%1$S” zostali wykluczeni przez %2$S.
-message.banMaskRemoved=Użytkownicy łączący się z adresów pasujących do „%1$S” nie będą dłużej niedopuszczani przez %2$S.
+message.banMasks=Użytkownicy łączący się z następujących adresów są zbanowani z „%S”:
```
Comment (splewako) [inline chat/irc.properties]: I don't like [whole zbanowano change](https://github.com/piotrdrag/mozilla-l10n-pl/commit/706864f92fb935c7ec3a2b1dd26ac6e34295a8c2) and don't see merits for it.

File: chat/xmpp.properties [hunk 3]
```diff
@@ -46,6 +69,43 @@ chatRoomField.password=_Hasło
...
+
+conversation.message.parted.you=Pokój został opuszczony.
+conversation.message.parted.you.reason=Pokój został opuszczony: %S
+conversation.message.parted=%1$S opuścił pokój.
+conversation.message.parted.reason=%1$S opuścił pokój: %2$S
+
+conversation.message.invitationDeclined=%1$S odrzucił zaproszenie.
+conversation.message.invitationDeclined.reason=%1$S odrzucił zaproszenie: %2$S
+
+conversation.message.banned=%1$S został zbanowany z pokoju.
```
Comment (splewako) [inline chat/xmpp.properties]: As mentioned earlier, I don't like word "zbanowany".

File: chat/xmpp.properties [hunk 4]
```diff
@@ -46,6 +69,43 @@ chatRoomField.password=_Hasło
...
+conversation.message.banned.you.actor.reason=%1$S zbanował Cię z pokoju: %2$S
+
+conversation.message.kicked=%1$S został wyrzucony z pokoju.
+conversation.message.kicked.reason=%1$S został wyrzucony z pokoju: %2$S
+conversation.message.kicked.actor=%1$S wyrzucił %2$S z pokoju.
+conversation.message.kicked.actor.reason=%1$S wyrzucił %2$S z pokoju: %3$S
+conversation.message.kicked.you=Wyrzucono Cię z pokoju.
+conversation.message.kicked.you.reason=Wyrzucono Cię z pokoju: %1$S
+conversation.message.kicked.you.actor=%1$S wyrzucił Cię z pokoju.
+conversation.message.kicked.you.actor.reason=%1$S wyrzucił Cię z pokoju: %2$S
```
Comment (splewako) [inline chat/xmpp.properties]: Cię

File: chat/xmpp.properties [hunk 5]
```diff
@@ -46,6 +69,43 @@ chatRoomField.password=_Hasło
...
+conversation.message.invitationDeclined=%1$S odrzucił zaproszenie.
+conversation.message.invitationDeclined.reason=%1$S odrzucił zaproszenie: %2$S
+
+conversation.message.banned=%1$S został zbanowany z pokoju.
+conversation.message.banned.reason=%1$S został zbanowany z pokoju: %2$S
+conversation.message.banned.actor=%1$S zbanował %2$S z pokoju.
+conversation.message.banned.actor.reason=%1$S zbanował %2$S z pokoju: %3$S
+conversation.message.banned.you=Zbanowano Cię z pokoju.
+conversation.message.banned.you.reason=Zbanowano Cię z pokoju: %1$S
+conversation.message.banned.you.actor=%1$S zbanował Cię z pokoju.
```
Comment (splewako) [inline chat/xmpp.properties]: Cię

File: chat/xmpp.properties [hunk 6]
```diff
@@ -46,6 +69,43 @@ chatRoomField.password=_Hasło
...
+conversation.message.banned.you.actor.reason=%1$S zbanował Cię z pokoju: %2$S
```
Comment (splewako) [inline chat/xmpp.properties]: Cię

File: chat/xmpp.properties [hunk 7]
```diff
@@ -46,6 +69,43 @@ chatRoomField.password=_Hasło
...
+
+conversation.message.kicked=%1$S został wyrzucony z pokoju.
+conversation.message.kicked.reason=%1$S został wyrzucony z pokoju: %2$S
+conversation.message.kicked.actor=%1$S wyrzucił %2$S z pokoju.
+conversation.message.kicked.actor.reason=%1$S wyrzucił %2$S z pokoju: %3$S
+conversation.message.kicked.you=Wyrzucono Cię z pokoju.
```
Comment (splewako) [inline chat/xmpp.properties]: Cię

File: chat/xmpp.properties [hunk 8]
```diff
@@ -46,6 +69,43 @@ chatRoomField.password=_Hasło
...
+conversation.message.kicked.you.reason=Wyrzucono Cię z pokoju: %1$S
```
Comment (splewako) [inline chat/xmpp.properties]: Cię

File: chat/xmpp.properties [hunk 9]
```diff
@@ -46,6 +69,43 @@ chatRoomField.password=_Hasło
...
+conversation.message.kicked.you.actor=%1$S wyrzucił Cię z pokoju.
```
Comment (splewako) [inline chat/xmpp.properties]: Cię

File: chat/xmpp.properties [hunk 10]
```diff
@@ -46,6 +69,43 @@ chatRoomField.password=_Hasło
...
+conversation.message.kicked.you.actor.reason=%1$S wyrzucił Cię z pokoju: %2$S
+
+conversation.message.removedNonMember=%1$S został usunięty z pokoju, ponieważ jego konfiguracja została zmieniona na tylko dla członków.
+conversation.message.removedNonMember.actor=%1$S został usunięty z pokoju, ponieważ %2$S zmienił jego konfigurację na tylko dla członków.
+conversation.message.removedNonMember.you=Usunięto Cię z pokoju, ponieważ jego konfiguracja została zmieniona na tylko dla członków.
```
Comment (splewako) [inline chat/xmpp.properties]: Cię

File: chat/xmpp.properties [hunk 11]
```diff
@@ -46,6 +69,43 @@ chatRoomField.password=_Hasło
...
+conversation.message.removedNonMember.you.actor=Usunięto Cię z pokoju, ponieważ %1$S zmienił jego konfigurację na tylko dla członków.
```
Comment (splewako) [inline chat/xmpp.properties]: Cię

File: chat/irc.properties [hunk 12]
```diff
@@ -80,24 +78,24 @@ message.whoisEntry=\ua0\ua0\ua0\ua0%1$S: %2$S
...
+message.banMaskRemoved=Użytkownicy łączący się z adresów pasujących do „%1$S” nie są już zbanowani przez %2$S.
 message.ping=Odpowiedź ping od użytkownika %1$S po jednej sekundzie.;Odpowiedź ping od użytkownika %1$S po #2 sekundach.;Odpowiedź ping od użytkownika %1$S po #2 sekundach.
 
 error.noChannel=Nie ma takiego kanału jak „%S”.
 error.tooManyChannels=Przyłączenie do %S było niemożliwe - użytkownik jest już przyłączony do zbyt wielu kanałów.
 error.nickCollision=Pseudonim aktualnie wykorzystywany, zmienianie pseudonimu na „%1$S” [%2$S].
 error.erroneousNickname=„%S” jest niedozwolonym pseudonimem.
-error.banned=Zablokowano możliwość przyłączenia się do tego serwera.
-error.bannedSoon=Wkrótce nastąpi wykluczenie z serwera.
+error.banned=Zbanowano Cię z tego serwera.
```
Comment (splewako) [inline chat/irc.properties]: Cię

File: chat/irc.properties [hunk 13]
```diff
@@ -80,24 +78,24 @@ message.whoisEntry=\ua0\ua0\ua0\ua0%1$S: %2$S
...
+error.bannedSoon=Wkrótce nastąpi zbanowanie z serwera.
 error.mode.wrongUser=Nie można zmieniać trybów innym użytkownikom.
 error.noSuchNick=Użytkownik „%S” nie jest obecnie połączony.
 error.wasNoSuchNick=Nie było takiego użytkownika jak „%S”.
 error.noSuchChannel=Kanał „%S” nie istnieje.
 error.unavailable=Kanał „%S” jest chwilowo niedostępny.
-error.channelBanned=Masz „bana” na kanale „%S”.
+error.channelBanned=Zbanowano Cię z kanału „%S”.
```
Comment (splewako) [inline chat/irc.properties]: Cię

File: chat/xmpp.properties [hunk 14]
```diff
@@ -46,6 +69,43 @@ chatRoomField.password=_Hasło
...
+conversation.message.parted.reason=%1$S opuścił pokój: %2$S
+
+conversation.message.invitationDeclined=%1$S odrzucił zaproszenie.
+conversation.message.invitationDeclined.reason=%1$S odrzucił zaproszenie: %2$S
+
+conversation.message.banned=%1$S został zbanowany z pokoju.
+conversation.message.banned.reason=%1$S został zbanowany z pokoju: %2$S
+conversation.message.banned.actor=%1$S zbanował %2$S z pokoju.
+conversation.message.banned.actor.reason=%1$S zbanował %2$S z pokoju: %3$S
+conversation.message.banned.you=Zbanowano Cię z pokoju.
```
Comment (splewako) [inline chat/xmpp.properties]: Cię

File: chat/xmpp.properties [hunk 15]
```diff
@@ -46,6 +69,43 @@ chatRoomField.password=_Hasło
...
+conversation.message.banned.you.reason=Zbanowano Cię z pokoju: %1$S
```
Comment (splewako) [inline chat/xmpp.properties]: Cię

File: chat/xmpp.properties [hunk 16]
```diff
@@ -28,15 +28,38 @@ connection.error.failedToGetAResource=Pobieranie zasobu nie powiodło się
...
+conversation.error.joinForbidden=Nie można dołączyć do pokoju „%S”, ponieważ zbanowano Cię w nim.
 conversation.error.joinFailedNotAuthorized=Wymagana rejestracja: nie posiadasz uprawnień, aby dołączyć do tego pokoju.
 conversation.error.creationFailedNotAllowed=Ograniczony dostęp: nie posiadasz uprawnień do tworzenia pokojów.
 conversation.error.joinFailedRemoteServerNotFound=Nie można było dołączyć do pokoju „%S”, ponieważ serwer, na którym jest utrzymywany, jest nieosiągalny.
+conversation.error.changeTopicFailedNotAuthorized=Nie posiadasz uprawnień, aby ustawić temat tego pokoju.
+conversation.error.sendFailedAsNotInRoom=Nie można wysłać wiadomości do pokoju „%1$S”, ponieważ już w nim nie jesteś: %2$S
+conversation.error.sendFailedAsRecipientNotInRoom=Nie można wysłać wiadomości do pokoju „%1$S”, ponieważ adresat już w nim nie jest: %2$S
 conversation.error.remoteServerNotFound=Serwer adresata jest nieosiągalny
+conversation.error.nickNotInRoom=%S nie jest w pokoju.
+conversation.error.banCommandAnonymousRoom=Nie możesz banować uczestników z anonimowych pokojów. Zamiast tego spróbuj „/kick”.
```
Comment (splewako) [inline chat/xmpp.properties]: banować

File: chat/xmpp.properties [hunk 17]
```diff
@@ -62,3 +122,13 @@ odnoklassniki.protocolName=Odnoklassniki
 gtalk.usernameHint=adres e-mail
 
 odnoklassniki.usernameHint=Identyfikator profilu
+
+command.join3=%S [&lt;pokój&gt;[@&lt;serwer&gt;][/&lt;pseudonim&gt;]] [&lt;hasło&gt;]: Dołącza do pokoju, opcjonalnie podając inny serwer, pseudonim lub hasło pokoju.
+command.part2=%S [&lt;wiadomość&gt;]: Opuszcza pokój (z opcjonalną wiadomością).
+command.topic=%S [&lt;nowy temat&gt;]: Ustawia temat pokoju.
+command.ban=%S &lt;pseudonim&gt;[&lt;wiadomość&gt;]: Banuje uczestnika z pokoju. Tylko administrator pokoju może to robić.
```
Comment (splewako) [inline chat/xmpp.properties]: Banuje

File: chat/irc.properties (hunk 2 above)
Comment (piotrdrag) [inline chat/irc.properties]: It was inconsistent before (wykluczać vs. niedopuszczać vs. even some existing instances of banować). "Banować" is a part of the language now ([PWN](http://sjp.pwn.pl/szukaj/banowa%C4%87.html)) and is, in my opinion, much easier to understand for the users of any chat program. If you're staunchly against it, then what do you propose instead? I guess we could use "wykluczać".

File: chat/xmpp.properties (hunk 1 above)
Comment (piotrdrag) [inline chat/xmpp.properties]: What do you propose instead?

Comment (splewako) [inline chat/xmpp.properties]: "Ciebie" or (even better) nothing if possible.

"Nie można dołączyć do pokoju „%S”, ponieważ Ciebie w nim zbanowano." (ignoring discussion about zbanowano here)

Comment (piotrdrag) [inline chat/xmpp.properties]: Fixed.

File: chat/irc.properties (hunk 2 above)
Comment (splewako) [inline chat/irc.properties]: It is jargon, blind calque and PWN makes me laugh limiting this to chat only.

Single translation would be better but I could live with inconsistent usage of existing Polish words expressing it correctly and I also like "wykluczać" most.

Comment (piotrdrag) [inline chat/irc.properties]: "Wykluczać" it is then. Fixed.

**Comment by piotrdrag** [comment]
Should be all done now.

**Comment by splewako** [comment]
Thank you for the patches, landed as https://hg.mozilla.org/l10n-central/pl/pushloghtml?changeset=7271e7fa17cb and https://github.com/aviarypl/mozilla-l10n-central-pl/compare/f1d9cc2947c5...c8df04f3108a

**Pull request closed without merging**